### PR TITLE
Update Flevoland datasets (municipalities, province) to 2019

### PR DESCRIPTION
**Pull request by redekok**

### CODE REVIEW

File: datasets/PV24_flevoland/graph_values.yml [hunk 1]
```diff
@@ -122,67 +122,67 @@ transport_shipping_mixer_diesel:
...
   demand: 0.0
 transport_final_demand_electricity:
-  demand: 306.003548
+  demand: 464.54743199999996
 transport_final_demand_kerosene:
   demand: 0.0
 energy_power_ultra_supercritical_network_gas:
   demand: 0.0
-  full_load_hours: 4600.0
+  full_load_hours: 1590.0
```
Comment (redekok) [inline datasets/PV24_flevoland/graph_values.yml:146]: What's behind this significant decrease in full load hours? Is this due to the national update from 2018 to 2019?

File: datasets/PV24_flevoland/PV24_flevoland.derived.ad [hunk 2]
```diff
@@ -31,58 +31,58 @@
 - households_insulation_employment_constant = 0.014
 - co2_emission_1990_aviation_bunkers = 4.6046
 - co2_emission_1990_marine_bunkers = 34.95619
-- co2_percentage_free = 0.85
-- co2_price = 0.0078
+- co2_percentage_free = 0.4
+- co2_price = 0.02483
```
[the remark this answers is not in the document]
Comment (Charlottevm) [reply datasets/PV24_flevoland/PV24_flevoland.derived.ad:35]: I think this is a question regarding the general NL data update as these numbers were simply copied, @mabijkerk is there an explanation behind this?

File: datasets/GM0034_almere/GM0034_almere.derived.ad [hunk 3]
```diff
@@ -133,45 +133,45 @@
...
+- co2_emission_1990 = 0.363971453
 - coast_line = 10.93
 - interconnector_capacity = 60000.0
 - total_land_area = 129.24
-- number_of_buildings = 4538.0
-- number_of_cars = 254095.0
-- number_of_residences = 82312.0
-- number_of_inhabitants = 203990.0
-- offshore_suitable_for_wind = 12500.0
+- number_of_buildings = 2714.0
```
[the remark this answers is not in the document]
Comment (Charlottevm) [reply datasets/GM0034_almere/GM0034_almere.derived.ad:142]: This is weird, when I look at the source data from the municipality update and add all municipalities I get 9263 buildings in Flevoland? This suggests there is either something wrong with the data collection or with the amalgamator.. Ill check if this is also the case for the 2018 data!

[the remark this answers is not in the document]
Comment (Charlottevm) [reply datasets/GM0034_almere/GM0034_almere.derived.ad:142]: When adding municipalities in the 2018 data I get 9206.. which is compliant with the 2019 data but not with any of the data here. I am not sure what goes wrong but im guessing it has something to do with the update or amalgamator as the source data seems correct.

File: datasets/PV24_flevoland/graph_values.yml (hunk 1 above)
Comment (Charlottevm) [reply datasets/PV24_flevoland/graph_values.yml:146]: This is correct looking at the source-data and directly derived from the 2019 NL dataset. @mabijkerk do you have any idea?

Comment (Charlottevm) [reply datasets/PV24_flevoland/graph_values.yml:146]: According to @mabijkerk: The installed capcitiy of gas powerplants has remained the same but gas powerplants have produced less electricity in 2019 compared to 2015 which leads to a decrease in full load hours.

**Review by Charlottevm — APPROVED** [review]
The number of buildings and flh in 2018 are incorrect, however, all new 2019 numbers are correct so I would go ahead and merge!

**Review by lottevanvlimmeren — APPROVED** [review]

**Pull request merged**